prov/shm/src/smr_ep.c

```diff
@@ -347,15 +312,16 @@ static int smr_format_ipc(struct smr_cmd *cmd, void *ptr, size_t len,
 	cmd->msg.hdr.size = len;
 	cmd->msg.data.ipc_info.iface = iface;
 	cmd->msg.data.ipc_info.device = device;
-	ret = ofi_hmem_get_base_addr(cmd->msg.data.ipc_info.iface, ptr, len,
-				     &base,
-				     &cmd->msg.data.ipc_info.base_length);
+
+	ret = ofi_hmem_get_base_addr(cmd->msg.data.ipc_info.iface, ptr,
+				len, &base,
+				&cmd->msg.data.ipc_info.base_length);
 	if (ret)
 		return ret;
 
 	ret = ofi_hmem_get_handle(cmd->msg.data.ipc_info.iface, base,
-				   cmd->msg.data.ipc_info.base_length,
-				   (void **)&cmd->msg.data.ipc_info.ipc_handle);
+			cmd->msg.data.ipc_info.base_length,
+			(void **)&cmd->msg.data.ipc_info.ipc_handle);
 	if (ret)
```

Comment:
These format changes are unnecessary, and the original format was preferred.

Reply:
This refers to the changes in the previous 10 lines. Somehow the whole range was not marked.